cookbook/attributes/agent.rb

```diff
@@ -0,0 +1,6 @@
+default[:baragon][:group_name] = 'default'
+default[:baragon][:proxy_conf_dir] = '/tmp'
+default[:baragon][:upstream_conf_dir] = node[:baragon][:proxy_conf_dir]
```

Comment:
Defining attributes from other attributes makes wrapping this cookbook a bit tricky. I can't remember exactly what the correct pattern is, but memory suggests this is the one that might result in pain for someone wrapping this cookbook later & trying to only update `node['baragon']['proxy_conf_dir']`.